torch/nn/functional.py

```diff
@@ -2986,6 +2987,8 @@ def cross_entropy(
             of smoothing when computing the loss, where 0.0 means no smoothing. The targets
             become a mixture of the original ground truth and a uniform distribution as described in
             `Rethinking the Inception Architecture for Computer Vision <https://arxiv.org/abs/1512.00567>`__. Default: :math:`0.0`.
+        dim (int, optional): Specifies the dimension of the classes (the dimension across which softmax is computed).
+            Default: 1
```

Comment:
The default here is actually `None` ?

Reply:
@albanD, I set Default as 1 because I thought if no value is specified by the user for dim, then by default softmax would be computed across the first dimension.

Reply:
Yeah I think that is fine but we make these doc match 1-1 the implementation. You can say: the values default to `1` if not provided. Default: None.

Reply:
@albanD, Got it, made Default to ``None``